components/tools/ToolsList.js

```diff
@@ -1,12 +1,25 @@
 import ToolsCard from './ToolsCard';
 import Heading from '../typography/Heading'
+import { useRef, useEffect } from 'react';
 import Paragraph from '../typography/Paragraph'
 export default function toolsList({ toolsData }) {
+  const categoryRefs = useRef({});
+
+  useEffect(() => {
```

Comment:
Add a comment, explaining about this `useEffect` block.

Reply:
It scrolls the page to a specific category referenced in the URL hash.

Reply:
@akshatnema  Done Sir